### PR TITLE
Explicitly describe $ as an alias for jQuery

### PR DESCRIPTION
**Pull request by AurelioDeRosa**
Fixes gh-680

### CODE REVIEW

File: page/about-jquery/how-jquery-works.md [hunk 1]
```diff
@@ -53,6 +53,8 @@ $( document ).ready(function() {
 });
 ```
 
+<div class="warning">**Note**: The jQuery library exposes its methods and properties via two properties of the <code>window</code> object called <code>jQuery</code> and <code>$</code>. <code>$</code> is simply an alias for <code>jQuery</code> and it's often employed because it's shorter and faster to write.</div>
```
Comment (arthurvr) [inline page/about-jquery/how-jquery-works.md]: Why should this be in a `class="warning"` box? Might as well just have a normal paragraph about this. The warnings always give me the feeling something is a gotcha or something could go wrong, but in this case it's just a concept being introduced.

Comment (AurelioDeRosa) [inline page/about-jquery/how-jquery-works.md]: I feel it doesn't flow well with the rest of the discussion which has another focus, so I put it in a "aside" bar.

**Comment by AurelioDeRosa** [comment]
@agcolom should I merge it or should I address the comment of @arthurvr? I think it's fine as is but happy to update the PR:

**Comment by agcolom** [comment]
@AurelioDeRosa We could use either a normal note or a div class="note" which gives a blue box if I remember correctly.

**Comment by AurelioDeRosa** [comment]
@agcolom This is more of a general comment but I've checked the `note` class and it doesn't look very readable. I think the contrast isn't enough especially for `<code>`.

**Comment by agcolom** [comment]
@AurelioDeRosa I think you can land your PR :-)

**Comment by AurelioDeRosa** [comment]
Thank you for the review @agcolom. I'm going to merge it now.

**Pull request closed without merging**